### PR TITLE
Introduce 'Source code availability' section

### PR DESCRIPTION
**Pull request by pkoch**

### CODE REVIEW

**Review by hugopeixoto — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -101,7 +101,18 @@ The code (and any other produced artifacts) will belong to the participants. How
...
 In the interest of keeping things level, we’re only allowing to incorporate free/libre open source software that’s publicly available before the competition begins. We don’t have any issues with people using popular software packages, but we’d like to avoid that teams bring a whole project along when others couldn’t have done the same. If you have a non-trivial codebase you’d like to use on your hack, make sure it is publicly available under [an OSI approved license](https://opensource.org/licenses) and unencumbered by any IP restrictions before the competition begins. This way everybody starts equally as far from the finish line.
 
-At the beginning of the fair, we’ll make a public fork of your repo. This ensures that we’ll have a copy that we can make available to everyone and that we can preserve it for posterity.
+
+#### Source code availability
+
+In order to assure that all code is available to everyone, participants should
+use the GitHub repository provided by the organization. Nuking the repository
+and force pushes will be disabled. This ensures that we’ll have a copy that we
+can make available to everyone and that we can preserve it for posterity.
```
Comment (jpmcosta) [inline README.md]: Maybe: "and that we can preserve for posterity."

File: README.md [hunk 2]
```diff
@@ -101,7 +101,18 @@ The code (and any other produced artifacts) will belong to the participants. How
...
+
+Not using the repository **may disqualify their entire team** from winning any
+prizes.  If any team has a problem with this setup they must convene promptly
```
Comment (jpmcosta) [inline README.md]: Double space.

Comment (jpmcosta) [reply README.md]: Maybe: "If a team has any problem"

File: README.md [hunk 3]
```diff
@@ -101,7 +101,18 @@ The code (and any other produced artifacts) will belong to the participants. How
...
-At the beginning of the fair, we’ll make a public fork of your repo. This ensures that we’ll have a copy that we can make available to everyone and that we can preserve it for posterity.
+
+#### Source code availability
+
+In order to assure that all code is available to everyone, participants should
+use the GitHub repository provided by the organization. Nuking the repository
+and force pushes will be disabled. This ensures that we’ll have a copy that we
+can make available to everyone and that we can preserve it for posterity.
+
+Not using the repository **may disqualify their entire team** from winning any
```
Comment (nunopolonia) [inline README.md]: "will disqualify"

**Review by nunopolonia — APPROVED** [review]

**Pull request merged**